### PR TITLE
Disable pretty print

### PR DESCRIPTION
**Pull request by hythof**
Relate: https://github.com/WOVNio/equalizer/issues/9799

Disable pretty print for reduce html size.

### CODE REVIEW

File: src/main/java/com/github/wovnio/wovnjava/HtmlConverter.java [hunk 1]
```diff
@@ -18,22 +18,23 @@
     HtmlConverter(Settings settings, String original) {
         this.settings = settings;
         doc = Jsoup.parse(original);
+        doc.outputSettings().indentAmount(0).prettyPrint(false);
```
Comment (a-bougouin) [inline src/main/java/com/github/wovnio/wovnjava/HtmlConverter.java]: Is it needed to set `indentAmount`? Isn't it used for pretty printing only?

**Comment by a-bougouin** [comment]
Could you have a test that asserts original HTML indenting is preserved?

File: src/main/java/com/github/wovnio/wovnjava/HtmlConverter.java (hunk 1 above)
Comment (hythof) [reply src/main/java/com/github/wovnio/wovnjava/HtmlConverter.java]: I'm wrong. The option is for pretty printing only.
I removed it.
Thanks.

**Comment by hythof** [comment]
I added a test for pretty printing.

File: src/test/java/com/github/wovnio/wovnjava/HtmlConverterTest.java [hunk 2]
```diff
@@ -10,6 +10,14 @@
 
 public class HtmlConverterTest extends TestCase {
 
+    public void testDisablePrettyPrint() {
+        String original = "<html><head></head><body>\n" + "hello" + "\n</body></html>";
```
Comment (a-bougouin) [inline src/test/java/com/github/wovnio/wovnjava/HtmlConverterTest.java]: Could you add spacing before "hello"?:
```
String original = "<html><head></head><body>\n" + "  hello" + "\n</body></html>";
```

**Pull request merged**